src/GeositeFramework/sample_plugins/identify_point/main.js

```diff
@@ -197,6 +137,18 @@ define(["dojo/_base/declare", "framework/PluginBase"],
                     html: '<img src="' + imageSourcePath + '"/>',
                     fixed: true,
                 });
+            },
+
+            render: function(frameworkParameters) {
+                $(this.container).append(template);
```

Comment:
Ah, we should have done this from the start.

Reply:
Yeah, it makes it a little easier to manage the DOM.